Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/model/ConceptEntry.java

```diff
@@ -0,0 +1,308 @@
+        return word;
+    }
+    
+    public void setWord(String word) {
+        this.word = word;
+    }
+       
+    public String getDescription() {
+        return description;
+    }
+    
+    public void setDescription(String description) {
+        this.description = description;
+    }   
+    
+    public String getPos() {
+        return pos;
+    }
+   
+    public void setPos(String pos) {
+        this.pos = pos;
+    }
+    
+    public String getConceptList() {
+        return conceptList;
+    }
+  
+    public void setConceptList(String conceptList) {
+        this.conceptList = conceptList;
+    }   
+    
+    public String getSimilarTo() {
+        return similarTo;
+    }
+   
+    public void setSimilarTo(String similar) {
+        this.similarTo = similar;
+    }
+   
+    public String getSynsetIds() {
+        return synsetIds;
+    }
+       
+    public void setSynsetIds(String synsetIds) {
+        this.synsetIds = synsetIds;
+    }
+  
+    public void setModified(String modified) {
+        this.modified = modified;
+    }
+   
+    public String getModified() {
+        return modified;
+    }
+   
+    public boolean isDeleted() {
+        return isDeleted;
+    }
+   
+    public void setDeleted(boolean isDeleted) {
+        this.isDeleted = isDeleted;
+    }
+   
+    public List<ChangeEvent> getChangeEvents() {
+         return this.changeEvents;
+    }
+   
+    public void addChangeEvent(ChangeEvent changeEvent) {
+        if(this.changeEvents == null) {
+            this.changeEvents = new ArrayList<>();
+        }
+        this.changeEvents.add(changeEvent);
+    }
+   
+    public String getModifiedUser() {
+        return modifiedUser;
+    }
+    
+    public void setModifiedUser(String modifiedUser) {
+        this.modifiedUser = modifiedUser;
+    }
+    
+    public String getMergedIds() {
+        return mergedIds;
+    }
+       
+    public void setMergedIds(String mergedIds) {
+        this.mergedIds = mergedIds;
+    }
+    
+    public Set<String> getAlternativeIds() {
+        if(this.alternativeIds != null) {
+            String[] ids = this.alternativeIds.split(",");
+            return new HashSet<>(Arrays.asList(ids));
+        }
+        return new HashSet<>();
+    }
+    
+    public void addAlternativeId(String alternativeId) {
+        
+        if(alternativeId == null) return;
+        
+        if(this.alternativeIds == null || this.alternativeIds.isEmpty()) {
+            this.alternativeIds = alternativeId;
+        }else {
+            /*This loop is to handle alternativeId parameter with one or more alternativeIds*/
+            for(String altId : alternativeId.split(",")) {
+                altId = altId.trim();
+                this.alternativeIds = (altId.length() > 0 && checkDuplicate(altId)) ? this.alternativeIds : this.alternativeIds+","+altId;
+            }
+        }
+    }
+    
+    private boolean checkDuplicate(String alternativeId) {
```

Comment:
this should be called `isDuplicate` so that's its clear what it returns (true if it is a duplicate otherwise false). It's a lot harder to read and understand the above code if you have to look up what the method returns.